### ISSUE TITLE
rancher-webhook server No compatible with k8s v1.16.x

### ISSUE THREAD

**Issue by zhcsyncer** [rancher/rancher#29561]
**What kind of request is this (question/bug/enhancement/feature request):**

bug

**Steps to reproduce (least amount of steps as possible):**

deploy rancher-webhook

**Result:**

rancher-webhook pod logs

```
+ rancher-webhook-675ffbbbbb-9l49q › rancher-webhook
rancher-webhook-675ffbbbbb-9l49q rancher-webhook time="2020-10-16T06:02:58Z" level=info msg="Starting rbac.authorization.k8s.io/v1, Kind=Role controller"
rancher-webhook-675ffbbbbb-9l49q rancher-webhook time="2020-10-16T06:02:58Z" level=info msg="Starting rbac.authorization.k8s.io/v1, Kind=ClusterRoleBinding controller"
rancher-webhook-675ffbbbbb-9l49q rancher-webhook time="2020-10-16T06:02:58Z" level=info msg="Starting rbac.authorization.k8s.io/v1, Kind=ClusterRole controller"
rancher-webhook-675ffbbbbb-9l49q rancher-webhook time="2020-10-16T06:02:58Z" level=info msg="Starting rbac.authorization.k8s.io/v1, Kind=RoleBinding controller"
rancher-webhook-675ffbbbbb-9l49q rancher-webhook time="2020-10-16T06:02:58Z" level=info msg="Starting management.cattle.io/v3, Kind=RoleTemplate controller"
rancher-webhook-675ffbbbbb-9l49q rancher-webhook time="2020-10-16T06:02:58Z" level=info msg="Starting management.cattle.io/v3, Kind=GlobalRole controller"
rancher-webhook-675ffbbbbb-9l49q rancher-webhook time="2020-10-16T06:02:58Z" level=info msg="Starting rbac.authorization.k8s.io/v1, Kind=ClusterRole controller"
rancher-webhook-675ffbbbbb-9l49q rancher-webhook time="2020-10-16T06:02:58Z" level=info msg="Listening on :9443"
rancher-webhook-675ffbbbbb-9l49q rancher-webhook time="2020-10-16T06:02:58Z" level=info msg="Active TLS secret cattle-webhook-tls (ver=1463643067) (count 1): map[listener.cattle.io/cn-rancher-webhook:rancher-webhook listener.cattle.io/hash:e712b4a48ca6c6824c3041d1e7c8d8e87df685d692607ce91148d9382fd34e12]"
rancher-webhook-675ffbbbbb-9l49q rancher-webhook time="2020-10-16T06:02:58Z" level=info msg="Starting /v1, Kind=Secret controller"
rancher-webhook-675ffbbbbb-9l49q rancher-webhook time="2020-10-16T06:02:58Z" level=info msg="Sleeping for 15 seconds then applying webhook config"
rancher-webhook-675ffbbbbb-9l49q rancher-webhook time="2020-10-16T06:03:13Z" level=error msg="error syncing 'cattle-system/cattle-webhook-ca': handler secrets: failed to create rancher.cattle.io admissionregistration.k8s.io/v1, Kind=ValidatingWebhookConfiguration for  cattle-system/cattle-webhook-ca: ValidatingWebhookConfiguration.admissionregistration.k8s.io \"rancher.cattle.io\" is invalid: [webhooks[0].admissionReviewVersions: Invalid value: []string{\"v1\"}: must include at least one of v1beta1, webhooks[1].admissionReviewVersions: Invalid value: []string{\"v1\"}: must include at least one of v1beta1], requeuing"
rancher-webhook-675ffbbbbb-9l49q rancher-webhook time="2020-10-16T06:03:13Z" level=info msg="Active TLS secret cattle-webhook-tls (ver=1463643067) (count 1): map[listener.cattle.io/cn-rancher-webhook:rancher-webhook listener.cattle.io/hash:e712b4a48ca6c6824c3041d1e7c8d8e87df685d692607ce91148d9382fd34e12]"

```

**Other details that may be helpful:**

https://github.com/rancher/webhook/blob/25ffe2f1e89ba4d2a39467c8b0599d76d31d727f/pkg/admission/server.go#L101

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): v2.5.1
- Installation option (single install/HA): HA

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported): Imported
- Machine type (cloud/VM/metal) and specifications (CPU/memory):
- Kubernetes version (use `kubectl version`):

```
Client Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.3", GitCommit:"06ad960bfd03b39c8310aaf92d1e7c12ce618213", GitTreeState:"clean", BuildDate:"2020-02-13T18:08:14Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"16+", GitVersion:"v1.16.9-aliyun.1", GitCommit:"4f7ea78", GitTreeState:"", BuildDate:"2020-05-08T07:29:59Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
```

- Docker version (use `docker version`):

```
(paste the output here)
```

**Comment by superseb** [comment]
Same as https://github.com/rancher/rancher/issues/29420

**Comment by StrongMonkey** [comment]
@dramich Can you cut a new release of rancher-webhook?

**Comment by dramich** [comment]
v0.1.0-beta7 is building.... https://github.com/rancher/webhook/releases/tag/v0.1.0-beta7

**Comment by StrongMonkey** [comment]
Available to test on 2.5-head and master-head

**Comment by sowmyav27** [comment]
**Verified on 2.5-head commit id: `52583ea7a0` and master-head commit id: `e4dc25135d`**

- Deploy HA rke with local cluster k8s version - `v1.16.15`
- Rancher webhook is seen dpeloyed successfully
- Logs of the helm operation pod installing rancher-webhook and the rancher webhook pod do not have any errors
- Logs of  helm operation pod installing rancher-webhook
```
Waiting for Kubernetes API to be available
--
Tue, Oct 20 2020 6:08:42 am | helm upgrade --history-max=5 --install=true --namespace=cattle-system --reset-values=true --timeout=5m0s --values=/home/shell/helm/values-rancher-webhook-0.1.0-beta700.yaml --version=0.1.0-beta700 --wait=true rancher-webhook /home/shell/helm/rancher-webhook-0.1.0-beta700.tgz
Tue, Oct 20 2020 6:08:42 am | Release "rancher-webhook" does not exist. Installing it now.
Tue, Oct 20 2020 6:08:45 am | creating 5 resource(s)
Tue, Oct 20 2020 6:08:45 am | beginning wait for 5 resources with timeout of 5m0s
Tue, Oct 20 2020 6:08:47 am | Deployment is not ready: cattle-system/rancher-webhook. 0 out of 1 expected pods are ready
Tue, Oct 20 2020 6:08:49 am | Deployment is not ready: cattle-system/rancher-webhook. 0 out of 1 expected pods are ready
Tue, Oct 20 2020 6:08:52 am | NAME: rancher-webhook
Tue, Oct 20 2020 6:08:52 am | LAST DEPLOYED: Tue Oct 20 00:38:43 2020
Tue, Oct 20 2020 6:08:52 am | NAMESPACE: cattle-system
Tue, Oct 20 2020 6:08:52 am | STATUS: deployed
Tue, Oct 20 2020 6:08:52 am | REVISION: 1
Tue, Oct 20 2020 6:08:52 am | TEST SUITE: None
Tue, Oct 20 2020 6:08:52 am |  
Tue, Oct 20 2020 6:08:52 am | ---------------------------------------------------------------------
Tue, Oct 20 2020 6:08:52 am | SUCCESS: helm upgrade --history-max=5 --install=true --namespace=cattle-system --reset-values=true --timeout=5m0s --values=/home/shell/helm/values-rancher-webhook-0.1.0-beta700.yaml --version=0.1.0-beta700 --wait=true rancher-webhook /home/shell/helm/rancher-webhook-0.1.0-beta700.tgz
```
